connect.yaml

```diff
@@ -24,9 +24,5 @@ deployAs:
         - key: CTP_CLIENT_SECRET
           description: Commercetools client secret
   - name: enabler
-    applicationType: service
-    endpoint: /
-    configuration:
-      standardConfiguration:
-        - key: CTP_REGION
-          description: Region of Commercetools project
\ No newline at end of file
+    applicationType: assets
+    endpoint: /
```

Comment:
thanks @leungkinghin , we can remove endpoint too.

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/111/commits/7f0656f959c2711fe25576dbd6e1a0314445b6c8